alexandrie/src/main.rs

```diff
@@ -83,14 +86,22 @@ async fn run() -> Result<(), Error> {
 
     info!("running database migrations");
     #[rustfmt::skip]
-    state.repo.run(|conn| embedded_migrations::run(conn)).await
+        state.repo.run(|conn| embedded_migrations::run(conn)).await
```

Comment:
I don't think this line should be more indented than the rest of the body.
```suggestion
    state.repo.run(|conn| embedded_migrations::run(conn)).await
```

Reply:
Before that,I just exec cmd:`cargo fmt`. That I find `#[rustfmt::skip]` it's so closed with `info`log.
I just add new line fixed this question. Then exec cmd:`cargo fmt`. It's ok